dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -286,4 +286,81 @@ tlsProfiler.option-desc.file=\
 tlsProfiler.option-key.verbose=\
 \ \ \ \ --verbose
 tlsProfiler.option-desc.verbose=\
-\tInclude additional information about unsafe protocols.
\ No newline at end of file
+\tInclude additional information about unsafe protocols.
+
+# -------- CONFIGURE FIPS OPTIONS --------------------------------------#
+# List of descriptive text for options. Use "\n\" for line continuations.
+# The code provides no whitespace: begin keys with "\ \ \ \ " for leading
+# whitespace, and begin descriptions with \t 
+#------------------------------\n at 72 chars -------------------------\n\#
+#------------------------------\n at 72 chars Leading "\ \ \ \ "-------\n\#
+#------------------------------\n at 72 chars Leading "\t"-------------\n\# 
+configureFIPS.desc=\
+\tConfigure FIPS 140-3 based on the Java installation configured by the\n\
+\tJAVA_HOME, JRE_HOME, or WLP_DEFAULT_JAVA_HOME environment variables\n\
+\tin that order. If none was configured, FIPS 140-3 will be configured\n\
+\tbased on the JVM used to the securityUtility command. The command\n\
+\twill setup FIPS 140-3 for either IBM Semeru Runtimes or IBM SDK, or\n\
+\treturn an error if neither could be determined.\n\n\
+\tFor IBM Semeru Runtimes:\n\
+\tCreates a file containing the FIPS 140-3 custom profile for use by\n\
+\tyour application, which extends the Liberty FIPS140-3-Liberty profile\n\
+\twhich extends the IBM Semeru Runtimes FIPS140-3-Strongly-Enforced\n\
+\tprofile. Also, creates or updates the default.env, server.env, or\n\
+\tclient.env file to enable FIPS 140-3 by setting the ENABLE_FIPS140_3\n\
+\tenvironment variable to the path to the custom profile file.\n\n\
+\tFor IBM SDK:\n\
+\tCreates or updates the default.env, server.env, or client.env file to\n\
+\tenable FIPS 140-3 by setting the ENABLE_FIPS140_3 environment\n\
+\tvariable to true.
+
+configureFIPS.usage.options=\
+\t{0} configureFIPS [options]
+
+configureFIPS.option-key.1.server=\
+\ \ \ \ --server=name
+
+configureFIPS.option-desc.1.server=\
+\tServer for which to configure FIPS 140-3. If this value is not\n\
+\tprovided and no client is specified, then FIPS 140-3 will be\n\
+\tconfigured for all clients, servers, and commands.
+
+configureFIPS.option-key.2.client=\
+\ \ \ \ --client=name
+
+configureFIPS.option-desc.2.client=\
+\tClient for which to configure FIPS 140-3. If this value is not\n\
+\tprovided and no server is specified, then FIPS 140-3 will be\n\
+\tconfigured for all clients, servers, and commands.
+
+configureFIPS.option-key.3.customProfileFile=\
+\ \ \ \ --customProfileFile=name
+
+configureFIPS.option-desc.3.customProfileFile=\
+\tThe file in which to write the IBM Semeru Runtimes FIPS 140-3 custom\n\
+\tprofile. If this value is not provided, then the file will be created\n\
+\twith the name of FIPS140-3-Liberty-Application.properties in the WLP\n\
+\tinstallation's etc directory. If the server or client option was\n\
+\tused, the file will be created in the specified server's or client's\n\
```

Comment:
Change
the file will be created in the specified server's or client's security/resources directory.
to
the file is created in the security/resources directory for the specified server or client.